lte/gateway/configs/control_proxy.yml

```diff
@@ -39,14 +39,3 @@ proxy_cloud_connections: True
 
 # Allows http_proxy usage if the environment variable is present
 allow_http_proxy: False
-
```

Comment:
Let's keep these in place because
1. We have people who use it still
2. We still want to be able to override the values on a per-agw basis. (This is probably useful while we're testing out all the Sentry enablement changes. Once we can confirm we've gotten rid of noise and any performance risks, we can just have the configuration be all or nothing per-network.)

So I would modify the PR to always take the control_proxy value as a prescendence if it's not empty during this migration phase.

Reply:
@themarwhal The PR now reads the config from control_proxy.yml first (if sentry_url is present) :)